classes/sota_m3ulcb.bbclass

```diff
@@ -1,8 +1,8 @@
 # Commit united image to OSTree, not just uImage
 OSTREE_KERNEL = "Image"
 
-EXTRA_IMAGEDEPENDS_append_sota = " m3ulcb-ota-bootfiles"
-IMAGE_BOOT_FILES_sota += "m3ulcb-ota-bootfiles/*"
+EXTRA_IMAGEDEPENDS_append_sota = " renesas-ota-bootfiles"
+IMAGE_BOOT_FILES_sota += "renesas-ota-bootfiles/*"
```

Comment:
will this still be overridden by meta-agl/meta-agl-bsp/conf/include/agl_m3ulcb.inc?

Reply:
If it is duplicated in `agl_m3ulcb.inc` it should be removed from there. PR to AGL is coming after this one is merged.